torch/nn/modules/activation.py

```diff
@@ -714,7 +714,7 @@ def __init__(self, embed_dim, num_heads, dropout=0., bias=True, add_bias_kv=Fals
         self.num_heads = num_heads
         self.dropout = dropout
         self.head_dim = embed_dim // num_heads
-        assert self.head_dim * num_heads == self.embed_dim, "embed_dim must be divisible by num_heads"
+        assert self.head_dim * num_heads == self.embed_dim, "embed_dim = {} must be divisible by num_heads = {}".format(embed_dim, num_heads)
```

Comment:
Need to fix the flake8 error.

Reply:
should I change it to something like,
```
if self.head_dim * num_heads != self.embed_dim:
                    raise ValueError((
                       "embed_dim = {} must be divisible by num_heads = {}".format(embed_dim, num_heads)
```

Reply:
I think the line is just too long.

Reply:
it still give error

Reply:
Can you point me the error? It seems the failed test case is irrelevant.

Reply:
I meant it fails for one check caffe2 onnx ort2

Reply:
That one is from the upstream and it's irrelevant with your PR, IMO.

Reply:
then it should be fine